core/modules/dashboard/includes/block.news.inc

```diff
@@ -40,13 +40,30 @@ class DashboardNewsBlock extends Block {
    */
   public function getContent() {
     $news = $this->getNews();
+    $days_before_news_is_old = 14;
+
     $links = array();
     if ($news) {
       foreach ($news as $key => $notifications) {
-        $links[] = '<p>' . l(
-            $notifications['title'],
-            $notifications['url']
-          ) . ' - ' . filter_xss($notifications['summary']) . '</p>';
+        $notification_date = format_date($notifications['timestamp'], 'custom', 'M j, Y');
+        $date_classes = array('class' => array('notification-date'));
+        $is_new = (time() - $notifications['timestamp']) < $days_before_news_is_old * 24 * 60 * 60;
+        if ($is_new) {
+          $date_classes['class'][] = 'marker';
```

Comment:
I don't think we should use the `marker` class here. That has a specific meaning in other places, and is maybe not specific enough for this use. How about `recent` or even `notification-recent` ?